.github/workflows/delete-preview.yml

```diff
@@ -19,8 +17,7 @@ jobs:
         shell: bash
 
       # Delete the branch!
-      - name: Delete branch
-        uses: dawidd6/action-delete-branch@v3
-        with:
-          github_token: ${{ secrets.GH_PAT }}
-          branches: preview-${{ github.event.pull_request.number }}
+      - name: Delete branch locally and remotely
```

Comment:
I noticed that the canned github action `dawidd6/action-delete-branch@v3` has been archived, so we probably should move away from using that and just this the old fashioned way.